documentation/man/features/build.rst

```diff
@@ -148,3 +154,7 @@ If you want to clean your kernel::
 If you want to reset the kernel tree to its default, `all config and script output will be removed`::
 
   kw b --full-cleanup
+
+If you want to use cflags::
+
+  kw b --cflags "-DMY_BEAUTIFUL_FLAG"
```

Comment:
In general, it is better to add a real-world example. Add a more complex command, something with multiple flags.